algorithms/CPlusPlus/Arrays/Largest and Smallest Number in an array.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
+//simple approach:
+//sort the array in ascending order.
+//the first element would be the smallest and the last elemnet would be the largest
```

Comment:
I don't know how it actually went under the codespell's radar, but spelling of element is wrong here.

```suggestion
//the first element would be the smallest and the last element would be the largest
```